src/Components/fe-otp/FeOtp.js

```diff
@@ -36,27 +47,53 @@ export class FeOtp extends LocalizeMixin(LitElement) {
 
       return;
     }
+    this.checkOTP(ev);
 
-    const loginData = serializedValue;
-    this.dispatchEvent(
-      new CustomEvent('input-validation', { detail: loginData })
-    );
+    ev.target.reset();
+  }
+
+  async checkOTP(ev) {
+    this.isError = false;
+    try {
+      const { serializedValue } = ev.target;
+      await FeServices.postRequest({
+        url: '/otp',
+        data: {
+          code: serializedValue.otpcode,
+        },
+      });
+      this.dispatchEvent(
+        new CustomEvent('complete', { detail: serializedValue })
```

Comment:
why u are passing otp value ?, this component itself validate otp, parent need only completion status to proceed further

Reply:
Will remove the OTP value